### ISSUE TITLE
Update internal support pipeline

### ISSUE THREAD

**Issue by o-ram** [cal-itp/customer-success#390]
Goal: add more nuance to the "in progress" stage since some tickets take a long time.

- [x] Add on hold property back @laneymangan 
- [x] Add long term effort property back
- [x] Add custom properties to note where in progress things are (i.e. w/ data creation, maybe feed is created but we are waiting on trip planners) **see comment with list**
- [x] Add a blocked stage
- [x] Document changes in internal support pipeline guide

**Comment by o-ram** [comment]
Doc with labels that TDQ team uses: https://docs.google.com/document/d/1gJfcwaXBVQBthOpM9cE3VIYZRDircaBh4Rogd4t771g/edit#heading=h.howwqdn7tayl

**Comment by o-ram** [comment]
Idea for a custom property:
Data creation stage: 

- Initial interview
- feed creation
- hosting
- published
- trip planner
- live

**Comment by laneymangan** [comment]
@o-ram These are the options I'm seeing for utilizing the` on-hold` and `long term effort` ticket tags within the Internal Support Pipeline:
1. We keep the property "hidden", meaning it won't show up within the ticket unless you click "View all properties" and search for the property. When you then select a property from there, the tag will appear on the card view. This is probably the least efficient option, but it ensures that the TDQ team won't be confused (since having that property visible previously caused some confusion, which was why I ultimately removed it) 
2. We rename the "Support Ticket Subject" property (which is where on-hold and long-term effort are selected from) to something like "Ticket Details". This would change for both ticket pipelines. We can then make this property visible within the ticket view and hopefully, it won't cause any confusion. It will not be included in the "create a ticket" view. 

Thoughts?

**Comment by laneymangan** [comment]
I added `Data Creation Stage` property. I've added it to the "Internal Support Ticket Details" section. I've also added it to the card view. 

The property is a multi-select field including the options below. (I wasn't sure if any of these stages could overlap, I can switch it to single-select if that's better). The dropdown options can easily be edited if we need to remove/add any other stages. Let me know if you have any thoughts on this @o-ram and @kathrynmsullivan 

- Initial interview
- feed creation
- hosting
- published
- trip planner
- live

FYI, we are limited to 4 visible properties in the card view. The current card view properties are: 

- Create date
- Close date
- Ticket Owner
- Data Creation Stage

If we want to add another property like this, we can likely remove "close date" from the card view. 

<img width="261" alt="image" src="https://github.com/cal-itp/customer-success/assets/107429473/f3779452-62a9-4f42-b3a9-fe206718aaa1">

<img width="345" alt="image" src="https://github.com/cal-itp/customer-success/assets/107429473/1cc940fc-1b19-409d-82b8-5ca0733c62bf">

**Comment by o-ram** [comment]
> @o-ram These are the options I'm seeing for utilizing the` on-hold` and `long term effort` ticket tags within the Internal Support Pipeline:
> 
> 1. We keep the property "hidden", meaning it won't show up within the ticket unless you click "View all properties" and search for the property. When you then select a property from there, the tag will appear on the card view. This is probably the least efficient option, but it ensures that the TDQ team won't be confused (since having that property visible previously caused some confusion, which was why I ultimately removed it)
> 2. We rename the "Support Ticket Subject" property (which is where on-hold and long-term effort are selected from) to something like "Ticket Details". This would change for both ticket pipelines. We can then make this property visible within the ticket view and hopefully, it won't cause any confusion. It will not be included in the "create a ticket" view.
> 
> Thoughts?

I lean in favor of option 2 if there's no opposition to renaming. The name "Ticket Details" makes sense to me and as long as we keep the pipeline guide updated, hopefully nobody gets confused (especially since it wouldn't be in the "create ticket" view)

**Comment by o-ram** [comment]
> I added `Data Creation Stage` property. I've added it to the "Internal Support Ticket Details" section. I've also added it to the card view.
> 
> The property is a multi-select field including the options below. (I wasn't sure if any of these stages could overlap, I can switch it to single-select if that's better). The dropdown options can easily be edited if we need to remove/add any other stages. Let me know if you have any thoughts on this @o-ram and @kathrynmsullivan
> 
> * Initial interview
> * feed creation
> * hosting
> * published
> * trip planner
> * live
> 
> FYI, we are limited to 4 visible properties in the card view. The current card view properties are:
> 
> * Create date
> * Close date
> * Ticket Owner
> * Data Creation Stage
> 
> If we want to add another property like this, we can likely remove "close date" from the card view.
> 
> <img alt="image" width="261" src="https://private-user-images.githubusercontent.com/107429473/300084093-f3779452-62a9-4f42-b3a9-fe206718aaa1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYzMTI2MTUsIm5iZiI6MTcwNjMxMjMxNSwicGF0aCI6Ii8xMDc0Mjk0NzMvMzAwMDg0MDkzLWYzNzc5NDUyLTYyYTktNGY0Mi1iM2E5LWZlMjA2NzE4YWFhMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNlQyMzM4MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MzcwYjFmOTA5NmM2YjliYjdlNWU1MGVlYTE4MThkM2Q4YjI1YWQ0OGE2OTg0ZjIwMjNmODNkMDU3NTUwYmIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vw1VX_UseoKXwySpx443NNnobPb0Vv_-g5KTxNFJkt4"> <img alt="image" width="345" src="https://private-user-images.githubusercontent.com/107429473/300084201-1cc940fc-1b19-409d-82b8-5ca0733c62bf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYzMTI2MTUsIm5iZiI6MTcwNjMxMjMxNSwicGF0aCI6Ii8xMDc0Mjk0NzMvMzAwMDg0MjAxLTFjYzk0MGZjLTFiMTktNDA5ZC04MmI4LTVjYTA3MzNjNjJiZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNlQyMzM4MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDJkMmY2NTFiNTY0MWRjYTJlZjExZjhjZjgwNWFhNDU1ZDQ3NGQwYWY5YmQ5OWNjNzA0OGM2M2MxMGZlMmUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yl5wi3zxYd8eMGPf7oIMm5e2I6TABgLUP0WHjF08NUs">




**Property select options:** the stages make sense to me, but I ultimately defer to @kathrynmsullivan on what makes sense for tracking/displaying where in the data creation process an agency is.

**Card view properties:** agreed, if we need one to go now or at any time, "close date" is not critical to have in this view.

**Comment by kathrynmsullivan** [comment]
@laneymangan - the data creation stage makes sense to me. Thank you!

**Comment by laneymangan** [comment]
@o-ram When I'm going through tickets I filter for "open tickets assigned to me", it would be nice to filter out tickets that don't require my attention anymore (ie when the TDQ team has stepped in to either schedule a meeting or answer more technical questions) So I'm wondering if it would make sense to do one of the following:

1. Create a property or additional stage for when the TDQ team has stepped in and is either scheduling a meeting or answering more technical questions

2. Assign the ticket to the TDQ team member once they are the main person communicating with the agency
Cons: could potentially slip through the cracks since I won't see it as "my" ticket any more

Any thoughts? This definitely isn't absolutely necessary, but it might help when we have a lot of tickets to sort through

**Comment by laneymangan** [comment]
Update: `Support Ticket Subject` has been renamed `Ticket Details` and added to Internal Support Ticket Details conditional section. 

To note: this pulls in all of the property options (payments, GTFS, connectivity etc) even though we will only be utilizing `On Hold` and `Long Term Effort`, I don't see a way around this but since this is an internal CS team workflow I think it's fine. 

cc @o-ram

**Comment by o-ram** [comment]
> @o-ram When I'm going through tickets I filter for "open tickets assigned to me", it would be nice to filter out tickets that don't require my attention anymore (ie when the TDQ team has stepped in to either schedule a meeting or answer more technical questions) So I'm wondering if it would make sense to do one of the following:
> 
> 1. Create a property or additional stage for when the TDQ team has stepped in and is either scheduling a meeting or answering more technical questions
> 2. Assign the ticket to the TDQ team member once they are the main person communicating with the agency
>    Cons: could potentially slip through the cracks since I won't see it as "my" ticket any more
> 
> Any thoughts? This definitely isn't absolutely necessary, but it might help when we have a lot of tickets to sort through

Definitely can see how this would be useful! I am leaning towards **option 1** because of the same concern you highlighted for option 2. We could easily have a simple property that notes that the TDQ team has stepped in to provide technical assistance. Whatever makes sense to you for the structure of that property is fine with me!

**Comment by o-ram** [comment]
> Update: `Support Ticket Subject` has been renamed `Ticket Details` and added to Internal Support Ticket Details conditional section.
> 
> To note: this pulls in all of the property options (payments, GTFS, connectivity etc) even though we will only be utilizing `On Hold` and `Long Term Effort`, I don't see a way around this but since this is an internal CS team workflow I think it's fine.
> 
> cc @o-ram

Thank you so much, it looks great! And agreed, it's fine that all the property options have been pulled in - it's our workflow, and if it turns out to be a problem, we can deal with it then.

**Comment by laneymangan** [comment]
ok sounds good! I added a ticket property called "TDQ Support" with the options "Yes" or "No". Now we can filter for tickets assigned to us that aren't currently in the TDQ team's hands. I added the property under Internal Support Ticket Details. 



![Image](https://github.com/cal-itp/customer-success/assets/107429473/92d62d55-bde8-4fdb-bed2-e5d9082f2dc7)





![Image](https://github.com/cal-itp/customer-success/assets/107429473/070d4d98-e37b-46cb-848f-5708d6858a8a)
